README.md

```diff
@@ -1,8 +1,18 @@
 Randomizer
 ==========
 
+![screenshot](screenshot.png)
+
 Random symbol sequence (e.g. password) generator for Haiku
 
+Instructions:
+  * Clone the repo and `cd` to the folder where you cloned it and run the following:
+  ```
+  cd src
+  make
+  ./Randomizer
+  ```
+
```

Comment:
You can remove everything below. It's all outdated or has no informative contents.

Reply:
Removed the project webpage and changed the requirements.